### PR TITLE
2016Q4.feature.tools.camera_replay

### PR DESCRIPTION
**Pull request by bjornborg**

### CODE REVIEW

**Comment by chrberger** [comment]
@olbender I suggest that you review this PR.

File: code/tool/camera-replay/include/videocapture.hpp [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+	std::shared_ptr<odcore::wrapper::SharedMemory> m_sharedMemory;
+
+	std::string m_sourcename;
+	std::string m_filename;
+	uint32_t m_width;
+	uint32_t m_height;
+	uint32_t m_size;
+
+	std::unique_ptr<cv::VideoCapture> m_capture;
+  cv::Mat m_image;
```
Comment (olbender) [inline code/tool/camera-replay/include/videocapture.hpp:80]: Indentation problem.

**Review by olbender — APPROVED** [review]
Please fix the indentation, but then it's ok.

**Comment by bjornborg** [comment]
It is now fixed, please pull it in pre-release when possible so it becomes available for use.

**Pull request merged**